### PR TITLE
Update micrometer to 1.0.5, fix broken tests/code and update doc

### PR DESCRIPTION
**Pull request by jotak**
In Influx registry, call `close` method instead of `stop` as it will release the executor.

### CODE REVIEW

**Comment by jotak** [comment]
Fixes #36

File: src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -323,6 +323,11 @@ Finally it is possible to filter the returned metrics from their base names:
 
 This section lists all the metrics generated by monitoring the Vert.x core tools.
 
+A word about naming: the different metric backends may have different conventions or rules for naming metrics.
```
Comment (tsegismont) [inline src/main/asciidoc/index.adoc]: Replace `A word about naming:` with `NOTE:`

Comment (tsegismont) [inline src/main/asciidoc/index.adoc]: Replace `A word about naming:` with `NOTE:`

Comment (jotak) [reply src/main/asciidoc/index.adoc]: Done

**Pull request merged**